docs/src/main/mdoc/typeclasses/applicativeerror.md

```diff
@@ -0,0 +1,180 @@
+
+```scala mdoc:silent
+import cats._
+import cats.implicits._
+
+def attemptDivideApplicativeError[F[_]](x: Int, y: Int)(implicit ae: ApplicativeError[F, String]): F[Int] = {
+   if (y == 0) ae.raiseError("divisor is error")
+   else {
+      ae.pure(x/y)
+   }
+}
+```
+
+The above method summons `ApplicativeError` to provide behavior representing
+an error where the end-user, based on type, will get their 
+appropriate response.
+
+`ApplicativeError` is an `Applicative`, which means all `Applicative` 
+functions are available for use. One such method is `pure`, which will 
+return the `F[_]` representation, where `F` could represent `Either`. 
+
+Another method that you will see is `raiseError`, which will generate 
+the specific error type depending on what `F[_]` represents. 
+If `F[_]` is an `Either`, then `ae.raiseError` will return `Left`. 
+If `F[_]` represents a `Validation`, then `ae.raiseError` will return `Invalid`.
+For example, if we want to use an `Either` as our error representation,
+we can do the following:
+
+```scala mdoc:silent
+type OnError[A] = Either[String, A]
+val e: OnError[Int] = attemptDivideApplicativeError(30, 10)
+```
+
+or simply via assignment
+
+```scala mdoc:silent
+val f: Either[String, Int] = attemptDivideApplicativeError(30, 10)
+```
+
+### `Validated`
+
+Given the same function `attemptDivideApplicativeError`, we can call that
+function again but with a different return type, since the `ApplicativeError`
+can support other "error" based types.  Here we will use `cats.data.Validated`
+when calling `attemptDivideApplicativeError`. Notice that 
+`attemptDivideApplicativeError` is the same as we defined above, 
+so we make no other changes.
+
+```scala mdoc:silent
+import cats.implicits._
+import cats.data.Validated
+
+type MyValidated[A] = Validated[String, A]
+val g = attemptDivideApplicativeError[MyValidated](30, 10)
+```
+
+We can inline the right projection type alias, `MyValidated`, doing the following:
+
+```scala mdoc:silent
+val h = attemptDivideApplicativeError[({ type T[A] = Validated[String, A]})#T](30, 10)
+```
+
+Or we can use [KindProjector](https://github.com/typelevel/kind-projector) 
+to make this more refined and readable
+
+```scala mdoc:silent
+val j = attemptDivideApplicativeError[Validated[String, *]](30, 10)
+```
+
+## It is an `Applicative` after all
+
+As a Reminder, this is an [`Applicative`](/cats/typeclasses/applicative.html) 
+so all the methods of `Applicative` are available to you to use in 
+manipulating your values, `ap`, `mapN`, etc. In the following example, notice
+we are using `Applicative`'s `map2`, and of course, `pure` which also is a 
+form of `Applicative`.
+
+```scala mdoc:silent
+import cats.implicits._
+def attemptDivideApplicativeErrorWithMap2[F[_]](x: Int, y: Int)(implicit ae: ApplicativeError[F, String]): F[_] = {
+   if (y == 0) ae.raiseError("divisor is error")
+   else {
+     val fa = ae.pure(x)
+     val fb = ae.pure(y)
+     ae.map2(fa, fb)(_ + _)
```

Comment:
Stylistic question: do people prefer using `map2`? Ubiquitously in codebases I've worked on, it's unused, and instead people use the tuple ops, `(fa, fb).mapN(_ + _)`, which works for any number of arguments (no need to use `map2` vs `map3` vs `map4`, so on)

Reply:
I was thinking of `MonadError` as a separate page just purely for the reason so that it fits on the left border of the website. Also, the original `Applicative` and `Monad` were introduced separately.  I can try to do `Applicative and Monad Error`.  I could go either way, TBH. ;)  As per `map2`, I just chose one of the `Applicative` methods. I can change it to be more realistic.

Reply:
I'll try to include both on the same page with the menu and title of "Applicative and Monad Error"

Reply:
I'd be happy to have two separate pages and have them link to each other.